README.md

```diff
@@ -14,10 +14,44 @@ You can obtain [JSqlServerBulkInsert] from Maven by adding the following:
 <dependency>
 	<groupId>de.bytefish</groupId>
 	<artifactId>jsqlserverbulkinsert</artifactId>
-	<version>1.5</version>
+	<version>2.0</version>
 </dependency>
 ```
 
+
+## Supported Types ##
+
+Please read up the Microsoft Documentation for understanding the mapping between SQL Server Types and JDBC Data Types:
+
+* (Understanding the JDBC Driver Data Types)[https://docs.microsoft.com/en-us/sql/connect/jdbc/understanding-the-jdbc-driver-data-types]
+
+The following JDBC Types are supported by the library:
```

Comment:
Nice addition

Reply:
Yes, it's a little confusing how the types map. This may help.